src/Todo.js

```diff
@@ -19,13 +19,29 @@ function Todo({text,todo,todos,setTodos}) {
          }))
      }
 
+
+     // check Handler
+     const checkHandler = (e) => {  
+         console.log(todo.id)
+           setTodos(todos.map(item =>{
```

Comment:
Do we need to loop over the complete todos here?  Can't we use the index and update the check?